modules/Emails/EmailsController.php

```diff
@@ -183,8 +183,14 @@ public function action_GetFromFields()
         $ie = new InboundEmail();
         $ie->email = $email;
         $accounts = $ieAccountsFull = $ie->retrieveAllByGroupIdWithGroupAccounts($current_user->id);
-        $emailSignatures = unserialize(base64_decode($current_user->getPreference('account_signatures', 'Emails')));
-        $defaultEmailSignature = $current_user->getPreference('signature_default');
+        $accountSignatures = $current_user->getPreference('account_signatures', 'Emails');
+        if($accountSignatures != null) {
+            $emailSignatures = unserialize(base64_decode($accountSignatures));
+            $defaultEmailSignature = $current_user->getPreference('signature_default');
+        } else {
+            $defaultEmailSignature = null;
+            $GLOBALS['log']->warn('User has not signature');
```

Comment:
User has not signature?  You Mean 'no'  Should this be a warning at all as its up the user if they want one or not - not exactly a warning needed.